addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/RelativePathConfusionScanRule.java

```diff
@@ -247,7 +247,10 @@ public void scan() {
                 try {
                     hackedMessage.setCookieParams(originalMsg.getCookieParams());
                 } catch (Exception e) {
-                    log.warn("Could not set the cookies from the base request: {}", e);
+                    log.warn(
+                            "Could not set the cookies from the base request: {}",
+                            e.getMessage(),
+                            e);
```

Comment:
It was not adding the exception stack trace and looking at the core code this will not throw any (half the code is not catching any).